android/assets/jsons/Civ V - Vanilla/Units.json

```diff
@@ -27,6 +27,7 @@
 		"cost": 25,
 		"obsoleteTech": "Scientific Theory",
 		"uniques": ["Ignores terrain cost"],
+		"specialUpgradesTo": "Archer",
```

Comment:
This should be a unique - "Can upgrade to [unitType] via random through [Ancient Ruins]" (I tried thinking of a version that was ruins-replacement-agnostic but could not 😢 )